cmd/xl-v1-utils.go

```diff
@@ -77,6 +79,18 @@ func reduceQuorumErrs(errs []error, ignoredErrs []error, quorum int, quorumErr e
 	default:
 		// No quorum satisfied.
 		maxErr = errors2.Trace(quorumErr, errs...)
+
+		// Log errors from disks to help admin take action.
+		var diskErrLogs []interface{}
+		var errCount int
+		for i, diskErr := range errs {
+			if diskErr != nil {
+				diskErrLogs = append(diskErrLogs,
+					fmt.Sprintf("disk [%d/%d]: %v", i+1, len(errs), diskErr.Error()))
```

Comment:
As part of changes in https://github.com/minio/minio/pull/5450 , we have decided to use static strings and map to represent the values( in this case, the disk number and disk error).

Reply:
@kannappanr OK. In that case let's block this PR until error logging changes are merged.